.dumi/pages/index/components/Theme/ThemePicker.tsx

```diff
@@ -88,7 +88,7 @@ export default function ThemePicker(props: ThemePickerProps) {
   const [locale] = useLocale(locales);
 
   return (
-    <Space size={token.paddingLG}>
+    <Space size={token.paddingLG} style={{ flexWrap: 'wrap' }}>
```

Comment:
Just use the wrap property of Space.
![image](https://github.com/ant-design/ant-design/assets/10286961/ea45d96a-9178-4a8f-8dc7-91019d932a4c)

Reply:
done